### PR TITLE
feat: Enable light bridge on mainnet

### PR DESCRIPTION
**Pull request by wsdt**
## Changes
- Enabled light bridge on mainnet (hiding fast bridging)
- Minor refactoring

Merge once we want to go live with light bridge (everything ready aside from liquidity). 

@toser2

### CODE REVIEW

File: src/containers/earn/__snapshots__/Earn.test.tsx.snap [hunk 1]
```diff
@@ -279,7 +279,7 @@ exports[`Earn  should match snapshot on switching between tabs on L1 1`] = `
       </div>
       <span>
         <button
-          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMd MuiButton-containedSizeMd MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMd MuiButton-containedSizeMd MuiButton-colorPrimary css-vsfgff-MuiButtonBase-root-MuiButton-root"
+          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMd MuiButton-containedSizeMd MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMd MuiButton-containedSizeMd css-vsfgff-MuiButtonBase-root-MuiButton-root"
```
Comment (wsdt) [inline src/containers/earn/__snapshots__/Earn.test.tsx.snap:282]:
```suggestion
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMd MuiButton-containedSizeMd MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMd MuiButton-containedSizeMd MuiButton-colorPrimary css-vsfgff-MuiButtonBase-root-MuiButton-root"
```

File: src/containers/earn/__snapshots__/Earn.test.tsx.snap [hunk 2]
```diff
@@ -545,7 +545,7 @@ exports[`Earn  should match snapshot on switching between tabs on L2 1`] = `
...
```
Comment (wsdt) [inline src/containers/earn/__snapshots__/Earn.test.tsx.snap:548]:
```suggestion
 class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMd MuiButton-containedSizeMd MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMd MuiButton-containedSizeMd MuiButton-colorPrimary css-vsfgff-MuiButtonBase-root-MuiButton-root"
```

File: src/containers/home/__snapshots__/index.test.tsx.snap [hunk 3]
```diff
@@ -248,11 +248,6 @@ exports[`Home  should match snapshot 1`] = `
           >
             ©2023 Enya Labs
           </p>
```
Comment (wsdt) [inline src/containers/home/__snapshots__/index.test.tsx.snap:250]:
```suggestion
          </p>
           <p
            class="sc-imWYAI dZiXLh sc-gFVvzn VtWau"
          >
            v0.9.1
          </p>
```

**Review by sk-enya — APPROVED** [review]
Looks Good!

**Comment by sk-enya** [comment]
Are we going to release light bridge mainnet support with next gateway release this week?

**Comment by wsdt** [comment]
> Are we going to release light bridge mainnet support with next gateway release this week?

I would say the next one after (since we still wait for the liquidity, final approval/Go & Jyoti's pipeline)

**Pull request merged**